README.md

```diff
@@ -1,5 +1,164 @@
-# FLEDGE Polyfill
+# FLEDGE Shim
 
 We're building a pure-JavaScript implementation of the [FLEDGE
-spec](https://github.com/WICG/turtledove/blob/master/FLEDGE.md), built on top of
-existing browser APIs.
+spec](https://github.com/WICG/turtledove/blob/master/FLEDGE.md), on top of
+existing browser APIs. The goal is to allow testing as much of FLEDGE as
+possible, in as realistic a manner as possible, given the constraint of not
+being able to add new features to the browser itself.
+
+## Status
+
+This project is just beginning.  Don't expect anything to work yet!
+
+## Design
+
+FLEDGE requires a way to store information in the browser that is (a) accessible
+across all websites but (b) only through JavaScript access control.
+`localStorage` in a cross-origin iframe fits this well. In Chrome this is not
+partitioned and only JavaScript running within the iframe can read or modify the
+data.
+
+The shim is divided into two pieces:
+
+* A *binary* that runs in a cross origin iframe, such as
+  `https://fledge-shim.example/v/1234`.
+
+* A *library* that consumers compile into their code, and which communicates
```

Comment:
Consumers could also just add the tag to their page which wouldn't be compiling into their code. I'd just say: 

A *library* that runs on the top level page, and which communicates...

Reply:
done